tsconfig.json

```
@@ -1,23 +1,13 @@
 {
+  "extends": "@mui/monorepo/tsconfig.json",
   "compilerOptions": {
```

I've started to experience this issue in VSCode:
> To enable project-wide JavaScript/TypeScript language features, exclude large folders with source files that you do not work on.

I've experienced it in `material-ui` repository, but never in `material-ui-x`.
I've found this suggestion: https://github.com/microsoft/TypeScript/issues/12750#issuecomment-265806038 and adding proper excludes seem to solve the issue for me:

```suggestion
  "exclude": ["**/.*/*", "**/build/*", "**/node_modules/*", "docs/export/*"],
  "compilerOptions": {
```

Let's try it and if it works then we can move it upstream

---

Spoke too soon I guess - the issue is still there :/

---

Using this for now - seems to work better

```suggestion
  "exclude": ["**/.*/**/*", "**/build/**/*", "**/node_modules/**/*", "docs/export/**/*"],
  "compilerOptions": {
```